### PR TITLE
Add global trajectory parameters

### PR DESCRIPTION
**Pull request by ErikQQY**
Fix: #37

### CODE REVIEW

File: src/struct_named_trajectory.jl [hunk 1]
```diff
@@ -287,6 +292,7 @@ function NamedTrajectory(
         Z.final,
         Z.goal,
         Z.components,
+        z.params,
```
Comment (aarontrowbridge) [inline src/struct_named_trajectory.jl:295]: looks like this just needs to be upper case

**Comment by aarontrowbridge** [comment]
hey @ErikQQY the tests failed, looks like a variable should be uppercase, check the comment I left. It looks good otherwise tho! Let's get the tests passing and we'll merge it in.

File: src/struct_named_trajectory.jl (hunk 1 above)
Comment (ErikQQY) [reply src/struct_named_trajectory.jl]: yeah, already fixed

**Comment by aarontrowbridge** [comment]
CI is running, last thing @ErikQQY, can you add a docs file in line with the last bullet in the issue #41. :) looks good otherwise 👍

**Comment by ErikQQY** [comment]
The documentations has been added

**Comment by ErikQQY** [comment]
bump, if here are still something missing?

**Review by aarontrowbridge — COMMENTED** [review]
everything looks good!

**Review by aarontrowbridge — APPROVED** [review]
approving!

**Comment by aarontrowbridge** [comment]
@ErikQQY sorry, I caught something, and have been under the weather, it is merged in! Thanks so much!

**Comment by ErikQQY** [comment]
Wishing you all good! Take care!!!

**Pull request merged**